runtime/src/main/java/org/corfudb/runtime/collections/StreamingMap.java

```diff
@@ -21,4 +20,8 @@
      * @return stream of entries
      */
     Stream<Map.Entry<K, V>> entryStream();
+
+    default Stream<Map.Entry<K, V>> mutableEntryStream() {
```

Comment:
I think `unsafeStream` is a more appropriate adjective. It also makes it clear that it can only be used under an `Accessor` lock.

I would also prefer to not have a default method. Just stub it out in the implementation.